x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/anomaly_detection_panel.tsx

```diff
@@ -98,13 +98,14 @@ export const AnomalyDetectionPanel: FC = () => {
           return ml.results.getMaxAnomalyScore(group.jobIds, twentyFourHoursAgo, latestTimestamp);
         });
 
-      const results = await Promise.all(promises.map(p => p.catch(() => undefined)));
+      const results = await Promise.all(promises);
       const tempGroups = { ...groupsObject };
       // Check results for each group's promise index and update state
       Object.keys(scores).forEach(groupId => {
         const resultsIndex = scores[groupId] && scores[groupId].index;
-        scores[groupId] = resultsIndex !== undefined && results[resultsIndex];
-        tempGroups[groupId].max_anomaly_score = resultsIndex !== undefined && results[resultsIndex];
+        const { maxScore } = resultsIndex !== undefined && results[resultsIndex];
+        scores[groupId].maxScore = maxScore || undefined;
```

Comment:
@alvarezmelissa87 is this line needed?

Reply:
but as far as i can tell `tempGroups` on the line below is used for that check.
`scores` isn't used after this loop over its keys

Reply:
from my tests this line isn't needed, so i'm removing it.